CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+## [Upcoming] 2018-11-07
+1. Fix: crash on Patient Summary screen when a BP recorded earlier than six months is present.
```

Comment:
I've seen large projects maintain their changelog either using a CHANGELOG file or using Github releases. Do you think we should move to a file that's independent of the host?

Reply:
To me, this is sort of a "staging" area for release notes so that QA will know what to verify in this release. Once a release goes out, the detailed notes which are gathered in this file are updated on the GitHub release.

I wanted to use a GitHub release itself, but that creates a tag, and cherry picking commits would mean we have to update the tag when it actually goes live.

Do you have any other suggestion?